### ISSUE TITLE
Remove converter for afwa_snod2ioda.py and ims_scf2ioda.py

### ISSUE THREAD

**Issue by YoulongXia-NOAA** [JCSDA-internal/ioda-converters#882]
Update the converter identified in the title (hereafter referred to as the **target converter**) to follow the JEDI Data Conventions.

- These two ioda-converters input grib1/grib2 file 
- ioda-converters will not support such a data file as an input due to that pygrib is not well supported
- The land DA users have asked the developer to remove them
- clean land converters during the code sprint

## ZenHub organization

Please make sure that you have properly set:
- Labels: OBS3
- Estimate: An estimate of how long this issue will take to resolve. 1 = half a day. 2 = full day. 3 = two days, 5 = three days.
- Epics: [Sprint: Implement the JEDI Data Conventions in the Converters](https://github.com/JCSDA-internal/ioda-converters/issues/813)
- Assignees: Anyone **working** on this issue. Anyone who is not working directly on the issue, but
  who should receive notifications, should be listed below.
- Reviewers

## Notifications

Fill in a list of GitHub handles here. People here are not *assigned*, but instead receive notifications
about this issue's progress.

Notifications should always go to: @rhoneyager.

Other people:

**Comment by YoulongXia-NOAA** [comment]
@rhoneyager @srherbener, before I worked the other converters, let me to clean land converters first. As I talked for netCDF IMS snow cover data, it only has 4km snow cover data, I need to use python projection to produce latitude and longitude,  such a process will take a longer time and produce large size data. I do not have an idea how to handle it right now. I will work with land DA users and see if we can solve it. I will update it in the future.
